helm/templates/statefulset.yaml

```
@@ -0,0 +1,43 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: edc
+  labels:
+    app.kubernetes.io/name: edc
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: edc
+  serviceName: edc
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: edc
+    spec:
+      containers:
+        - name: edc
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: "{{ .Values.image.pullPolicy }}"
+          ports:
+            - containerPort: {{ .Values.container.port }}
+              name: http
+
+          readinessProbe:
+            initialDelaySeconds: 1
+            httpGet:
+              port: http
+              path: /api/check/readiness
+
+          livenessProbe:
+            initialDelaySeconds: 3
+            httpGet:
+              port: http
+              path: /api/check/liveness
+
+          startupProbe:
+            initialDelaySeconds: 10
+            failureThreshold: 7
+            httpGet:
+              port: http
+              path: /api/check/startup
```

```suggestion
              path: /api/check/readiness
```
See discussion at https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/pull/412

---

Ah thanks 👍 !